package.json

```diff
@@ -11,13 +11,14 @@
   "scripts": {
     "lint": "gts check",
     "lint:fix": "gts fix",
-    "start": "ts-node index.ts",
-    "test:build": "npm run compile:spectypes && tsc --noEmit",
-    "test": "npm run compile:spectypes && jest",
+    "start": "ts-node --compiler ttypescript index.ts",
```

Comment:
You already added a separate compile step for ts-runtime-checks, to me it looks that the transformer should not be required?

Using the `--compiler` argument would result in all libraries running though the ttypescript compiler. Would be good if we could avoid this.

Reply:
Removed.